PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.h

```diff
@@ -55,6 +55,10 @@ class AliFemtoDreamTrackCuts {
     fDCAPlots = plot;
   }
   ;
+  void SetTOFMass(bool mass) {
```

Comment:
More meaningful name e.g. SetPlotTOFMass would be good to not confuse it with a method to set a mass